### DISCUSSION TITLE
New to wezterm; how do I enable a "show/hide" hotkey binding?

### DISCUSSION THREAD

**Discussion by alan12345678** [wezterm/wezterm#3692] [Q&A]
Apologies if this is answered in the docs, but I couldn't find it.

I use a tiling window manager, so as such I sometimes want to run a quick command in a "floating" terminal window that I can show/hide, rather than have it open onto the desktop and squeeze whatever else is there.

Presently I use xfce4-terminal purely because it has a simple F12 "--drop down" config, allowing me to really quickly bring a terminal into focus and then send it back again. I'm happy to forego any drop down nonsense (wezterm looks lovely!) but can anyone please suggest a means whereby I can configure a simple key press to bring a wezterm window into focus, and then "unfocus" (not close it), allowing me it to overlay whatever I'm currently doing?

**Comment by wez** [comment]
Take a look at the discussion on:
* https://github.com/wez/wezterm/issues/1751
